client/src/app/site/polls/components/poll-progress/poll-progress.component.ts

```diff
@@ -41,12 +41,14 @@ export class PollProgressComponent extends BaseViewComponentDirective implements
                         map(users =>
                             /**
                              * Filter the users who would be able to vote:
-                             * They are present or have their right to vote delegated
+                             * They are present and don't have their vote right delegated
+                             * or the have their vote delegated to a user who is present.
                              * They are in one of the voting groups
                              */
                             users.filter(
                                 user =>
-                                    (user.is_present || user.isVoteRightDelegated) &&
+                                    ((user.is_present && !user.isVoteRightDelegated) ||
+                                        user.voteDelegatedTo?.is_present) &&
```

Comment:
This produces wrong results. You need to have the can_manage_user for user.voteDelegatedTo to return something. Determine the max value by (user.is_present || user.isVoteRightDelegated) is the correct way. The behavior is not to fix in the client at the moment.

Reply:
Works for me with perm can_see_user.

Reply:
most likely due to cached contents.

Reply:
> Works for me with perm can_see_user.

Sean is right there: without `can_manage_users` this does not work. BUT: The calculation @jwinzer is doing is correct! So we need to make sure, that poll managers (delegates cnnot see the progress) can see (not manage) the delegation structure (This is also good, since the delegation structure is tightly coupled to voting). I added a commit fixin this issue. THis is also approved by @emanuelschuetze .